### PR TITLE
Update Steering Committee Charter to reflect current project structure.

### PR DESCRIPTION
**Pull request by pwittrock**
Approvals: 9 needed to short-circuit lazy consensus

- [x] bgrant0607
- [ ] brendanburns
- [x] derekwaynecarr
- [x] dims
- [ ] jbeda
- [ ] michellN
- [x] philips
- [x] spiffxp
- [x] sarahnovotny
- [x] smarterclayton
- [x] timothysc
- [x] pwittrock

### CODE REVIEW

File: charter.md [hunk 1]
```diff
@@ -1,52 +1,108 @@
...
+be present in the relevant charters.  Additional delegated responsibilities 
+should also be specified in a charter document for the 
+party<sup>[2](#footnote2)</sup>.
+
+### SIG Architecture
+
+* Define, evolve, and defend the technical vision / mission and scope of the
+  project.
+* Decide which SIGs are responsible for each technical area of the 
+  project.  Chartering process still performed through the Steering Committee
```
Comment (smarterclayton) [inline charter.md]: Unclear sentence

**Comment by smarterclayton** [comment]
Looks pretty good on first pass. Nice and succinct.

**Comment by sarahnovotny** [comment]
looks solid.

File: charter.md [hunk 2]
```diff
@@ -1,52 +1,122 @@
...
+
+* Define, evolve, and defend a Code of Conduct, which must include a neutral,
+  unbiased process for resolving conflicts.
+* Respond to Code of Conduct Violations
+
+### All SIGs
+
+* Manage their own sub-projects, including creating new sub-projects, as
+  defined in their charters.
+* Create repositories for new sub-projects they own under sigs.k8s.io, as
```
Comment (dims) [inline charter.md]: `sigs.k8s.io` - would it be better to say `kubernetes-sigs github org`

**Comment by dims** [comment]
Looks great @pwittrock

**Comment by pwittrock** [comment]
@dims Updated

File: charter.md [hunk 3]
```diff
@@ -1,52 +1,124 @@
...
-  and to delegate appropriate authority those groups.
-* Define, evolve, and defend the vision, values, mission, and scope of the
-  project.
+* Provide top-level accountability for the health of the project.
+* Delegate management<sup>[1](#footnote1)</sup> to other
+  parties<sup>[2](#footnote2)</sup>.
+* Adapt the role and structure of the Steering Committee as needed to meet the
+  needs of the project.
+* Responsibilities not explicitly delegated to other
+  parties<sup>[2](#footnote2)</sup> either through this document or their
```
Comment (spiffxp) [inline charter.md]: reads strange to me with this order, maybe swap to: through their charters or this document?

should we document how we'd resolve conflict between the two? handled case-by-case? this document always takes precedence?

Comment (pwittrock) [reply charter.md]: Dropped the "this document or" so now it is just through their charters.

**Comment by spiffxp** [comment]
/approve
from me as well

/hold
so, how do we go about actually making this change?

https://github.com/kubernetes/steering/blob/master/changes.md - suggests a 4 week lead time and a 3/4 majority vote for "governance change for how the committee itself operates (e.g. charter, election process, etc.)".  It also says that "this process is intended to cover significant changes versus small tweaks."

Part of me feels like this isn't so much a change in the how the committee operates as a refactoring of the existing charter to reflect reality.  If I were making things up, I would give the community a week long RFC period via kubernetes-dev@ and seek a super-majority vote from steering as LGTM's on this PR once the period is over.  I don't know if I should be making things up though.

**Comment by pwittrock** [comment]
@spiffxp 

I agree we could fore a 4 week lead time and 3/4 vote.

Point of order - I believe the 4 week lead time is only for lazy consensus (does not require 3/4 vote), and a 3/4 vote can be called immediately.

This an update to our documentation to make it accurate, rather than a change in scope or bylaws of the Steering Committee, so 1/2 seems plenty.  This also isn't really an urgent change, so I am fine with chasing down 3/4 lgtms on this if we aren't in agreement about the acceptance process.

We have version control to track changes so I don't know what the risk here is.

File: charter.md [hunk 4]
```diff
@@ -1,52 +1,124 @@
...
-* Define and evolve project governance structures and policies, including how
-  contributors become committers/maintainers, approvers, reviewers, members,
-  etc.
+  transparency and accountability policies for such groups.
+  
+### Not yet delegated responsibilities
+
+The following responsibilities may be delegated in the future.
+
+* Manage the Kubernetes brand to decide which things can be called “Kubernetes”
```
Comment (timothysc) [inline charter.md]: We don't have ownership or enforcement here, that resides with the CNCF, but we can coordinate with them.

File: charter.md [hunk 5]
```diff
@@ -1,52 +1,124 @@
...
+  project resources/assets<sup>[5](#footnote5)</sup>  
+
+## Delegated responsibilities of note
+
+Following are noteworthy responsibilities that have been delegated and *should
+be present in the relevant charters*.  Additional delegated responsibilities 
+should also be specified in a charter document for the 
+party<sup>[2](#footnote2)</sup>.
+
+### SIG Architecture
```
Comment (timothysc) [inline charter.md]: This really does not belong in the charter imo.

File: charter.md [hunk 6]
```diff
@@ -1,52 +1,124 @@
...
+  * Changes to or creation of SIG charters should still be overseen (approved)
+    by the Steering Committee.
+* Provide oversight pertaining to which sub-projects created by SIGs are part
+  of the project.  If necessary, direct sub-projects outside the scope of the 
+  Kubernetes project to be moved outside the project.
+* Define what a conformant Kubernetes cluster is as part of any certification
+  process.
+  * Facilitated through the conformance Working Group.
+
+### SIG Release
```
Comment (timothysc) [inline charter.md]: Same.

File: charter.md [hunk 7]
```diff
@@ -1,52 +1,124 @@
...
-* Declare a release, so that the committee can ensure quality/feature/other
-  requirements are met.
-* Request funds and other support from the CNCF (e.g. marketing, press, etc.)
\ No newline at end of file
+* Control access to and establish processes regarding release related project
+  resources/assets, including artifact repositories, build and test
+  infrastructure, etc.
+* Declare a release and ensure quality/feature/other requirements are met.
+
+### Code of Conduct Committee
```
Comment (timothysc) [inline charter.md]: same.

**Comment by pwittrock** [comment]
@timothysc SGTM.

**Comment by bgrant0607** [comment]
cc @jdumars

File: charter.md [hunk 8]
```diff
@@ -1,52 +1,124 @@
...
+  parties<sup>[2](#footnote2)</sup> through their charters reside with
+  the Steering Committee.
+
+## Direct responsibilities of the Steering Committee
+
+The following responsibilities belong to the Steering Committee.
+
+* Act as escalation point of last resort for decisions important to the 
+  project that cannot be resolved within SIGs or Committees.
+* Delegate project responsibilities and ownership to SIGs and Committees.
```
Comment (bgrant0607) [inline charter.md]: We should be explicit that the intent is to delegate decisions on all technical matters to the appropriate SIG

File: charter.md (hunk 4 above)
Comment (bgrant0607) [reply charter.md]: I agree

File: charter.md (hunk 5 above)
Comment (bgrant0607) [reply charter.md]: I agree.

File: charter.md [hunk 9]
```diff
@@ -1,52 +1,124 @@
...
+Following are noteworthy responsibilities that have been delegated and *should
+be present in the relevant charters*.  Additional delegated responsibilities 
+should also be specified in a charter document for the 
+party<sup>[2](#footnote2)</sup>.
+
+### SIG Architecture
+
+* Define, evolve, and defend the technical vision / mission and scope of the
+  project.
+* Decide which SIG is responsible for each technical area of the
```
Comment (bgrant0607) [inline charter.md]: I think SIG Arch has only indirect influence over this, by
* deciding what is and is not in scope for the project as a whole
* deciding where some functionality belongs in the system architecturally

File: charter.md (hunk 7 above)
Comment (bgrant0607) [reply charter.md]: Some of this content was in the Steering charter originally. It looks like Phil was trying to be clear about to which subgroups those responsibilities were delegated. The alternative is to be vague here (e.g., don't mention code of conduct at all). Maybe we could categorize high-level areas, such as technical areas, policies, processes, etc., and just state that they should be delegated to subgroups, either SIGs or committees.

**Comment by pwittrock** [comment]
Took another swing at this.

File: charter.md [hunk 10]
```diff
@@ -1,52 +1,78 @@
...
+* Through the chartering review process, delegate ownership of, responsibility for
+  and authority over areas of the project to specific entities<sup>[2](#footnote2)</sup>.
+* Define, evolve, and defend the non-technical vision / mission and the values
+  of the project.
+* Charter and refine policy for defining new community groups<sup>[3](#footnote3)</sup>,
+  and establish transparency and accountability policies for such groups
+* Define and evolve project and group<sup>[3](#footnote3)</sup> governance
+  structures and policies<sup>[4](#footnote4)</sup>.
+* Act as a final non-technical escalation point for any Kubernetes repository<sup>[5](#footnote5)</sup>.
+* Request funds and other support from the CNCF (e.g. marketing, press, etc.)
```
Comment (timothysc) [inline charter.md]: @michelleN @spiffxp @dims - are there more details around requests that we want to explicitly call out here? 

IMO we should expand the scope of requests.

**Comment by spiffxp** [comment]
If it's less that 5K and it's a service desk ticket it doesn't ened to be part of the charter

File: charter.md (hunk 10 above)
Comment (philips) [reply charter.md]: decided to make this part of a separate PR on the steering call.

**Comment by smarterclayton** [comment]
lgtm as well

**Comment by bgrant0607** [comment]
Has enough check marks
/hold cancel

**Pull request merged**